orangecontrib/text/widgets/owconcordance.py

```diff
@@ -151,6 +155,21 @@ def matching_docs(self):
         else:
             return 0
 
+    def get_data(self):
+        domain = Domain([], metas=[StringVariable("Conc. {}".format(
+            self.word)), StringVariable("Document")])
+        data = []
+        docs = []
+        for row in range(self.rowCount()):
+            data.append([])
+            docs.append([])
```

Comment:
Not necessary to first append empty table and then override it. Just append the final one.

Reply:
Then this data[row].append(str(self.data(index))) won't work. How do I fill the table?

Reply:
Create a temporary table to append `str(self.data(index))` and then append `[" ".join(tmp)]` to `data`. For `docs` you can directly append `[self.corpus.titles[self.word_index[row][0]]]` to it.